src/VisualStudio/IntegrationTest/IntegrationTests/AbstractIntegrationTest.cs

```diff
@@ -86,7 +87,7 @@ protected virtual void Dispose(bool disposing)
             {
                 try
                 {
-                    _visualStudioContext.Dispose();
+                    _visualStudioContext?.Dispose();
```

Comment:
Do we know what changed to cause the context to become null here, where it wasn't previously?

Reply:
➡️ This was the cause of the "166 tests failed" issue seen in several builds recently. An error occurred during `InitializeAsync`, leading to a `NullReferenceException` here and subsequent failure to dispose of `_messageFilter`. The failure to dispose of the message filter ensured all subsequent tests would fail.